README.md

```diff
@@ -6,3 +6,6 @@ added requirements.txt
 -->Css
 -->React
 --> Handlebars
+
```

Comment:
I have added JavaScript, HTML5 and PHP as requirements.

[requirements.txt](https://github.com/inoblivion123/Hello-World/files/2522989/requirements.txt)